src/resource/link/test/link.resolver.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { LinkResolver } from '../link.resolver';
+import { LinkService } from '../link.service';
+
+describe('LinkResolver', () => {
+  let resolver: LinkResolver;
+
+  const mockLinkService = {};
```

Comment:
Is it the correct way for mocking?

Reply:
Definitely not, but I haven't written any of the tests yet. Planning to get the feature out first.